src/styles/block/contact.scss

```diff
@@ -0,0 +1,47 @@
+.contact {
+  margin: 0 auto;
+  margin-top: 150px;
+  overflow: hidden;
```

Comment:
The suggestion about images means to add it to the **direct** parent of each image